languages/python/openai_rllib/README.md

```diff
@@ -0,0 +1,86 @@
+# Create Anaconda environment
+
+This repo is a tutorial for installing and using OpenAI Gym on Eagle, as well as running experiments on single/multiple cores and nodes.
+
+Below are the basic steps for creating a dedicated Anaconda environment that you will use for all your experiments. Note that this environment will contain only some basic packages, but you can always augment it by installing further packages (please see at the bottom of this README file). See below for further details.
+
+If you have any questions, you can email us in the following addresses:
+* Kevin Sayers: Kevin.Sayers@nrel.gov
```

Comment:
I think we can drop my address as you are the expert. I would just point people to you anyways!

Reply:
Okay, that works although WEs asked me to put you as a collaborator on the repo, so I thought it good to put your address in the README as well. Anyway, I will remove it for now but if you want later, I will put it back.